po/CMakeLists.txt

```diff
@@ -1,3 +1,13 @@
 include (Translations)
 add_translations_directory ("${GETTEXT_PACKAGE}")
 add_translations_catalog ("${GETTEXT_PACKAGE}" ../src/ ../src/dbus-backend)
+
+find_package(Intltool REQUIRED)
```

Comment:
Everything above the `find_package()` call should be removed, as well as the `cmake/Translations.cmake` file from the repo. And the file should basically be the same as https://github.com/ubports/indicator-datetime/blob/xenial/po/CMakeLists.txt

Not having `ALL` argument on `intltool_update_potfile` is to prevent the file being updated with every build, resulting in conflicts when multiple people make changes, instead doing a manual rebuild of the pot file only when strings are changed.

Reply:
Ok. The problem ist just that many developers forget to do the pot update. We had already missing translations after new code came in. This is a QA issue of course but since our testers are also overlooking stuff the idea (like we have it in TELEports) is that the file is being generated every time, but should not be checked in except the changes are expected / do some real modifications. I am open for discussion. For this indicator it wont matter much though since its strings wont change very often.

Reply:
Can you make the requested changes @Flohack74 so we can merge this for OTA-14? Thanks.

Reply:
Ok will take a look...

Reply:
Any update on this, or should I just do it @Flohack74 ? Thanks.

Reply:
Done!